tests/parsers/test_pw2gw.py

```diff
@@ -27,7 +25,7 @@ def test_pw2gw_default(
     data_regression.check({'output_parameters': results['output_parameters'].get_dict()},
                           basename='test_pw2gw_default_data')
 
-    num_regression.check({k: v for k, v in results['eps'].get_iterarrays()}, basename='test_pw2gw_default_eps')
+    num_regression.check(results['eps'].get_iterarrays(), basename='test_pw2gw_default_eps')
```

Comment:
I think `.get_iterarrays()` is a generator - does `num_regression` iterate over it automatically? If not, this change might need reverting.

Reply:
wrapping in `dict()` should also work perhaps no?

Reply:
Absolutely.